src/solver/conv/conv_wino_fury_RxS.cpp

```diff
@@ -0,0 +1,415 @@
+                                                        const ProblemDescription& problem) const
+{
+    std::ignore = problem;
+
+    const bool coop_launch = ctx.GetStream().CooperativeLaunchSupported();
+    return coop_launch ? sync_buffer_size : 0; // 2KB buffer for global sync
+}
+
+template <uint32_t Winodata, uint32_t Winofilter>
+ConvSolution
+ConvWinoFuryRxS<Winodata, Winofilter>::GetSolution(const ExecutionContext& ctx,
+                                                   const ProblemDescription& problem) const
+{
+    const auto dev_name         = ctx.GetStream().GetDeviceName();
+    const auto cu_count         = ctx.GetStream().GetMaxHardwareComputeUnits();
+    const auto n_groups         = GetNGroups(cu_count);
+    const bool reduced_vgpr_mem = GpuHasReducedVGPRMem(dev_name);
+#if WORKAROUND_SWDEV_453577
+    const bool coop_launch = false;
+#else
+    const bool coop_launch = ctx.GetStream().CooperativeLaunchSupported();
+#endif
+
+    constexpr size_t wg_size = 384;
+
+    WinoShaderArgsV2 args;
+    // Main convolution parameters
+    if(!args.SetConvParams(problem))
+    {
+        MIOPEN_THROW(miopenStatusInternalError);
+    }
+
+    const auto shader_model = ShaderModel(ctx, args, cu_count, n_groups, reduced_vgpr_mem);
+    // For ASICs with redused VGPR memory we have only c16 kernel
+    const bool c32_mode = reduced_vgpr_mem ? false : shader_model.IsC32ModePreferable();
+
+    // Warning
+    static bool IsWarned = false;
+    if(!IsWarned)
+    {
+        if(cu_count != n_groups)
+        {
+            MIOPEN_LOG_WE(SolverDbId()
+                          << ": GPU has " << cu_count << " CUs, but this solver supports max "
+                          << n_groups << " and thus may show sub-optimal performance.");
+        }
+        IsWarned = true;
+    }
+
+    // Kernel name & file
+    const std::string kernel_version = "_v2_4_1";
+    std::string kernel_name          = "miopenSp3AsmConvFury" + kernel_version;
+    std::string kernel_file          = "Conv_Winograd_Fury" + kernel_version;
+
+    if(StartsWith(dev_name, "gfx11"))
+    {
+        kernel_name += "_gfx11";
+        kernel_name += reduced_vgpr_mem ? "_1024vgprs" : "_1536vgprs";
+    }
+    else
+    {
+        MIOPEN_THROW(miopenStatusInternalError);
+    }
+
+    std::string kernel_postfix;
+
+    if(problem.IsFp16())
+    {
+        kernel_postfix += "_fp16_fp16acc";
+    }
+    else
+    {
+        MIOPEN_THROW(miopenStatusInternalError);
+    }
+
+    kernel_postfix += IS2X3 ? "_f2x3" : "_f3x2";
+    kernel_postfix += c32_mode ? "_c32" : "_c16";
+    kernel_postfix += "_stride1";
+
+    kernel_name += kernel_postfix;
+    kernel_file += kernel_postfix + ".s";
+
+    // KernelInfo
+    KernelInfo kernel;
+
+#if !MIOPEN_USE_COMGR
+    KernelBuildParameters options{
+        {"ROCM_METADATA_VERSION", 5}, // For AmdgcnAssemble(...)
+    };
+    kernel.comp_options = options.GenerateFor(kbp::GcnAsm{});
+#endif
```

Comment:
@averinevg Adding these conditionals would lead to precompiled binary cache misses for this kernel because
- options are included in the KDB key
- we use offline compiler (and offline assembler) when prepare system KDB (distributed as precompiled binary cache)

I am going to revert this. Please let me know if you have something against.